stoservice/src/main/java/com/tutrit/stoservice/service/UserService.java

```diff
@@ -0,0 +1,21 @@
+package com.tutrit.stoservice.service;
+
+import com.tutrit.stoservice.bean.User;
+import com.tutrit.stoservice.repository.UserRepository;
+
+public class UserService {
+
+    private UserRepository userRepository;
+
+    public UserService(final UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public User saveUser(User user) {
+        return userRepository.save(user);
+    }
+
+    public void setUserRepository(UserRepository userRepository) {
```

Comment:
You don't have a default constructor, so it's not a necessary method, when you create a class you already set a value in the class field.